src/config/backends/extra-uEnv.ts

```diff
@@ -63,6 +63,7 @@ export class ExtraUEnv extends ConfigBackend {
 		return (
 			(deviceType.endsWith('-nano') ||
 				deviceType.endsWith('-nano-emmc') ||
+				deviceType.endsWith('-nano-2gb-devkit') |
```

Comment:
`jetson-nano-2gb-devkit-emmc` won't fit the pattern here since it ends with `devkit-emmc`
```suggestion
				deviceType.includes('-nano-2gb-devkit') |
```

Reply:
Hi @cywang117 and thank you for your review. The Nano 2GB devkit module only comes with sd-card support and no eMMC, so I marked it as unsupported in https://github.com/balena-os/balena-supervisor/pull/1822/commits/4ed98a08947aee2df019b1850aec6e2cbec48a6d

Reply:
If an eMMC variant of the 2GB module gets launched in the future we could do the switch then

Reply:
I see what you mean! Thanks for clarifying @acostach 

I'm going to approve later, because there are some issues with the GitHub actions & merge == deploy which are being investigated and I want to make sure this makes its way properly into prod.

Reply:
Thank you Christina!

Reply:
@acostach , the failing GitHub action logs show:
```
[balena-supervisor]  > tsc --project tsconfig.json

[balena-supervisor]  src/config/backends/extra-uEnv.ts(63,3): error TS2322: Type 'number | boolean' is not assignable to type 'boolean'.

[balena-supervisor]    Type 'number' is not assignable to type 'boolean'.

[balena-supervisor]  src/config/backends/extra-uEnv.ts(66,5): error TS2447: The '|' operator is not allowed for boolean types. Consider using '||' instead.
```

Would you mind changing the pipe to a `||` and then running `npm run lint` locally to verify it doesn't error?

Reply:
Thanks @cywang117 for spotting that typo